src/lib/keys/KeyManager.ts

```diff
@@ -519,17 +522,19 @@ class KeyManager {
       }
     }
     // Read in file buffer and signature
-    const fileBuffer = this.fileSystem.readFileSync(filePath);
-    const isVerified = await this.verifyData(fileBuffer, keyBuffer!);
-    return isVerified;
+    const signatureBuffer = this.fileSystem.readFileSync(filePath);
```

Comment:
Using `fs` in other parts of the code, then `fileSystem` here is confusing. Standardize on this. We should never really be using `fs` unless we are talking about VFS and EFS.

The actual `fs` should be supplied by a model on `Store`. That `Store` itself may then use node fs or other things. Cause later we need to be able to  abstract this.

Reply:
yeah this should be standardized in the next PR